tests/test_font.py

```diff
@@ -46,6 +46,44 @@ def test_FontArguments_setCollectionIndex(fontarguments):
     fontarguments.setCollectionIndex(0)
 
 
+@pytest.mark.parametrize(
+    "index, overrides",
+    [
+        (0, [(0, skia.ColorBLACK), (1, skia.ColorRED), (5, skia.ColorBLUE)]),
+        (1, []),
+        (2, None),
+    ],
+)
+def test_FontArguments_setPalette(fontarguments, index, overrides):
+    if overrides is None:
+        palette = skia.FontArguments.Palette(index)
+    else:
+        palette = skia.FontArguments.Palette(
+            index,
+            skia.FontArguments.Palette.Overrides(
+                [
+                    skia.FontArguments.Palette.Override(*override)
+                    for override in overrides
+                ]
+            ),
+        )
+
+    len_overrides = len(overrides) if overrides is not None else 0
+    assert palette.index == index
+    assert len(palette.overrides) == len_overrides
+    assert palette.overrideCount == len_overrides
+    assert repr(palette) == f"Palette(index={index}, overrideCount={len_overrides})"
```

Comment:
I would probably not have a repr test - since the `__repr__` method itself is somewhat a personal choice and subjected to changes later too.